packages/frameworks/src/frameworks.ts

```diff
@@ -2241,7 +2241,7 @@ export const frameworks = [
       },
     },
     dependency: 'nitropack',
-    getOutputDirName: () => Promise<string>,
+    getOutputDirName: () => 'public',
```

Comment:
Does this need to be `async` like the others?
```suggestion
    getOutputDirName: async () => 'public',
```